### ISSUE TITLE
Windows on ARM, Support?

### ISSUE THREAD

**Issue by sidd-kishan** [bobranten/Ext4Fsd#10]
Please let us know when can we have an ARM64 version for Windows on ARM OS.

**Comment by rikka0w0** [comment]
From my personal view, it won't need a lot of effort to compile binaries for ARM64. Hopefully the author have time and is willing to do this.

**Comment by JeromTWinL** [comment]
I'm also using windows 10 on Raspberry pi 3 , I'm voting for your issue @sidd-kishan

**Comment by bobranten** [comment]
The driver can be compiled for ARM64 but I don't know how to do a install program for ARM. Shall I publish driver binaries to begin with for manual install?

**Comment by JeromTWinL** [comment]
Ok

**Comment by kristibektashi** [comment]
>  I don't know how to do a install program for ARM.

No need, Windows on Arm supports x86 (32-bit) programs (but not drivers)

**Comment by bobranten** [comment]
On Tue, 7 Jun 2022, Kristibek wrote:

>>  I don't know how to do a install program for ARM.
>
> No need, Windows on Arm supports x86 (32-bit) programs (but not drivers)

Then the install program can be almost the same but contain an ARM driver, 
I think I can produce that so I will try.

Bo Brantén

**Comment by bobranten** [comment]
Now we have signed driver files for ARM/ARM64 for they who can do a manual install:
https://www.accum.se/~bosse/ext2fsd/0.70/signed/
I will try to make a install program for ARM/ARM64 later.
/Bo Branten

**Comment by bobranten** [comment]
I have started to try to build an install program for ARM. Has anyone managed to do a manual install on ARM that works?

**Comment by sidd-kishan** [comment]
https://www.accum.se/~bosse/ext2fsd/0.70/signed/

**Comment by bobranten** [comment]
Yes, I have the same experience on ARM I can only do manual install, if I 
do inf install it says signing error. I think ARM has higher requiremens 
so there has to be a signed cab file also? I will continue to investigate.

Bo Branten
